certified-connectors/CyberProof/readme.md

```diff
@@ -0,0 +1,8 @@
+# CyberProof
```

Comment:
@DavidMeu Please follow the [templates](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors) we have provided, and have the required information in your connector documentation files.

Reply:
@sriyen-msft 
Followed